### PR TITLE
KPV-2334:: Application contest new report

### PR DESCRIPTION
**Pull request by DavidSanFer555**
New report of application contest. Cinfa definition

### CODE REVIEW

File: grails-app/controllers/kuorum/politician/CampaignController.groovy [hunk 1]
```diff
@@ -589,6 +590,18 @@ class CampaignController {
         ] as JSON)
     }
 
+    def abstractSendProposalsReport(RestKuorumApiService.ApiMethod reportApiMethod) {
+        KuorumUserSession campaignUser = springSecurityService.principal
+        Long participatoryBudgetId = Long.parseLong(params.campaignId)
```
Comment (pverdu-kuorum) [inline grails-app/controllers/kuorum/politician/CampaignController.groovy]: The name of the variables are specific for the Campaign abstraction.

**Review by pverdu-kuorum — CHANGES_REQUESTED** [review]
Just rename some variables into your new abstract function which make the AJAX request.

File: grails-app/controllers/kuorum/politician/CampaignController.groovy [hunk 2]
```diff
@@ -592,14 +592,14 @@ class CampaignController {
 
     def abstractSendProposalsReport(RestKuorumApiService.ApiMethod reportApiMethod) {
         KuorumUserSession campaignUser = springSecurityService.principal
-        Long participatoryBudgetId = Long.parseLong(params.campaignId)
-        participatoryBudgetService.sendReport(campaignUser, participatoryBudgetId, reportApiMethod)
+        Long campaignId = Long.parseLong(params.campaignId)
+        participatoryBudgetService.sendReport(campaignUser, campaignId, reportApiMethod)
```
Comment (pverdu-kuorum) [inline grails-app/controllers/kuorum/politician/CampaignController.groovy]: Move to CampaignService to uncouple of ParticipatoryBudgetService

**Review by pverdu-kuorum — CHANGES_REQUESTED** [review]
Extract sendReport to Campaign service.

**Review by iduetxe — APPROVED** [review]

**Review by pverdu-kuorum — APPROVED** [review]

**Pull request merged**